velox/exec/GroupingSet.h

```diff
@@ -336,6 +345,11 @@ class GroupingSet {
   // First row in remainingInput_ that needs to be processed.
   vector_size_t firstRemainingRow_;
 
+  // In case of grouping key reordering, spilled data is first loaded into
```

Comment:
```
// In case of distinct aggregation without aggregates and the grouping key reordered, the spilled data ..., and is then projected for output.
```